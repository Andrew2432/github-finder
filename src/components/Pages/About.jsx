import React from 'react';
import { Container } from 'react-bootstrap';

const About = () => {
  return (
    <Container>
      <h1>About page</h1>
      <p>
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Quod repellat
        alias neque maxime nobis ad reprehenderit dolorum nemo porro, enim
        maiores quae blanditiis dolore ducimus fugit, non animi laudantium
        soluta, exercitationem aspernatur? Eligendi dolores quisquam quos
        doloribus explicabo harum facilis.
      </p>
    </Container>
  );
};

export default About;
