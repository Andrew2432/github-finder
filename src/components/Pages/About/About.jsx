import React from 'react';
import { Container } from 'react-bootstrap';

const About = () => {
  return (
    <Container data-testid="container">
      <h1 data-testid="heading">About page</h1>
      <p data-testid="desc">
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Quod repellat
        alias neque maxime nobis ad reprehenderit dolorum nemo porro, enim
        maiores quae blanditiis dolore ducimus fugit, non animi laudantium
        soluta, exercitationem aspernatur? Eligendi dolores quisquam quos
        doloribus explicabo harum facilis.
      </p>
    </Container>
  );
};

export default About;
