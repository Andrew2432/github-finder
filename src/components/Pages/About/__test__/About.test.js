import React from 'react';
import { render, cleanup } from '@testing-library/react';
import About from '../About';

afterEach(cleanup);

describe('About component', () => {
  const { asFragment, findByTestId } = render(<About />);
  it('Matches snapshot', () => {
    expect(asFragment(<About />)).toMatchSnapshot();
  });
  it('Contains the components', () => {
    expect(findByTestId('container')).toBeTruthy();
    expect(findByTestId('heading')).toBeTruthy();
    expect(findByTestId('desc')).toBeTruthy();
  });
});
