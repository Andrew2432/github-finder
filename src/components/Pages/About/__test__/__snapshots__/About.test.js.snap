// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component Matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="container"
    data-testid="container"
  >
    <h1
      data-testid="heading"
    >
      About page
    </h1>
    <p
      data-testid="desc"
    >
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Quod repellat alias neque maxime nobis ad reprehenderit dolorum nemo porro, enim maiores quae blanditiis dolore ducimus fugit, non animi laudantium soluta, exercitationem aspernatur? Eligendi dolores quisquam quos doloribus explicabo harum facilis.
    </p>
  </div>
</DocumentFragment>
`;
